lte/gateway/c/oai/tasks/s1ap/s1ap_mme_nas_procedures.c

```diff
@@ -721,177 +850,201 @@ void s1ap_handle_conn_est_cnf(
    * Start the outcome response timer.
    * * * * When time is reached, MME consider that procedure outcome has failed.
    */
-  //     timer_setup(mme_config.s1ap_config.outcome_drop_timer_sec, 0,
```

Comment:
Nice clean-up! Please remove the comments before this too.

Reply:
Resolved